misc/packaging/acrn-hypervisor.postinst

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#* Copyright (c) 2020 Intel Corporation
+# postinst script for acrn-hypervisor
+set -e
+
+
+echo "please choose <scenario> ,<board> ,<disk type>"
+
+echo "Senario is ->
```

Comment:
"scenario" is misspelled multiple times in this script

Reply:
change typo